snapshots/MinimalDelegationTest.json

```diff
@@ -1,6 +1,6 @@
 {
   "authorize": "210092",
   "revoke": "58203",
-  "validateUserOp_missingAccountFunds": "32862",
-  "validateUserOp_no_missingAccountFunds": "4178"
+  "validateUserOp_missingAccountFunds": "61864",
```

Comment:
double in gas?

Reply:
oh these gas estimates are messed up bc they don't use via-ir